src/main/java/se/sundsvall/messagingsettings/api/MessagingSettingsResource.java

```diff
@@ -0,0 +1,52 @@
+package se.sundsvall.messagingsettings.api;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.ok;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.ConstraintViolation;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+import se.sundsvall.messagingsettings.api.model.SenderInfoResponse;
+import se.sundsvall.messagingsettings.service.MessagingSettingsService;
+
+@Tag(name = "Messaging Settings")
+@RestController
+@Validated
+public class MessagingSettingsResource {
```

Comment:
Normally we add 500 and 400 ApiResponse on class level. As all resources can return these in one way or another generally. (500 is missing and 400 is defined on method)